src/sentry/eventstream/kafka/backend.py

```diff
@@ -180,7 +181,7 @@ def _send(
         asynchronous: bool = True,
         headers: MutableMapping[str, str] | None = None,
         skip_semantic_partitioning: bool = False,
-        event_type: EventStreamEventType = EventStreamEventType.Error,
+        event_type: str = EventStreamEventType.Error,
```

Comment:
You shouldn't need a cast here, the type will be an Enum member here, but if you need a string use `EventStreamEventType.Error.value`